src/vcpkg/commands.build.cpp

```diff
@@ -1588,7 +1588,12 @@ namespace vcpkg
         {
             if (!all_dependencies_satisfied)
             {
-                return {BuildResult::CascadedDueToMissingDependencies, std::move(missing_fspecs)};
+                return {BuildResult::CascadedDueToMissingDependencies,
+                        Util::fmap(std::move(missing_fspecs),
+                                   [](std::pair<PackageSpec, std::set<std::string>>&& missing_features) {
+                                       return FeatureSpec{std::move(missing_features.first),
```

Comment:
In particular, I'm not sure if putting the already `,`'d string into a FeatureSpec is OK.

Reply:
Ah https://github.com/microsoft/vcpkg-tool/blob/fd8aa4277faab8e2d715ed3a03098f345d9e0e7d/include/vcpkg/packagespec.h#L58-L59 says "single feature" so I'm going to change this from `FeatureSpec`